index.html

```diff
@@ -574,10 +574,10 @@ <h4><dfn data-lt="serviceEndpoint" data-dfn-type="dfn" id="serviceEndpoint">serv
     </section>
 
     <section>
-      <h3>Verification methods</h3>
+      <h3>Verification relationships</h3>
       <p>
-These properties pertain to parameters that can be used to independently verify
-a proof according to a particular method.
+These properties indicate the means by which the verification method in the value can
```

Comment:
Hmm when I read this it wasn't immediately clear to me. I think the property value indicates the "means" of verification, but the property name indicates the "purpose" of verification?

Reply:
Fixed, thanks @peacekeeper